x-pack/test/api_integration/apis/siem/events.ts

```diff
@@ -112,7 +112,7 @@ const eventsTests: KbnTestProvider = ({ getService }) => {
               const lastEventTime = resp.data.source.LastEventTime;
               expect(lastEventTime).to.eql({
                 __typename: 'LastEventTimeData',
-                lastSeen: '2019-02-19T23:26:50.002Z',
+                lastSeen: '2019-04-26T21:45:14.012Z',
```

Comment:
Very interesting... 😉 

Hopefully here soon we can isolate a `data.json.zip` per integration test so we don't have to keep doing this as we add more data for new tests.

Reply:
Agreed! I feel bad about changing these tests that have nothing to do with my table just because I added data. I might end up acclimating and one day changing a valid failing test.